ENIGMAsystem/SHELL/Widget_Systems/Win32/dialogs.cpp

```diff
@@ -481,7 +490,7 @@ string get_open_filename(string filter,string filename,string caption)
     if (filter[i] == '|') filter[i] = 0;
 
   char fn[MAX_PATH];
-  strcpy(fn, filename.c_str());
+  strcpy(fn, remove_slash(filename).c_str());
```

Comment:
To clarify, what is the behavior if you do *not* remove the slash, here?

Reply:
The dialog doesn't show at all and the function immediately returns. Robert merged my last pr and I said that was ok because neither of us anticipated people would run into this issue with the OPENFILENAME structure before we would fix it in a following pr which I already planned on doing at that time which is the point of this pr. Fundies was mocking me and making me look stupid in from of everyone about this, as if I didn't know it would break anything. I knew it would. I just thought it would get fixed before anyone would notice and Robert agreed.

Reply:
What a bizarre failure mode... leave it to Microsoft, I guess.